apps/dapp-console-api/src/models/entities.ts

```diff
@@ -0,0 +1,22 @@
+import { pgTable, timestamp, uuid, varchar } from 'drizzle-orm/pg-core'
+
+export enum EntityState {
+  ACTIVE = 'active',
+  DISABLED = 'disabled',
+}
+
+export const entities = pgTable('entities', {
```

Comment:
I think we also might need to create a index for privy did

Reply:
since, `privy_did` is unique, I believe that it will be automatically indexed